shared/petnames/names.go

```diff
@@ -0,0 +1,26 @@
+package petnames
```

Comment:
The current golang petnames library does not allow for an easy, deterministic selection of petnames. This adds the functionality as a small shared package.